### ISSUE TITLE
BUG probplot matplotlib 1.3.1

### ISSUE THREAD

**Issue by josef-pkt** [statsmodels/statsmodels#1407]
I'm getting 8 failures like the following `TypeError: There is no line property "other"` 
with python 3.3 and matplotlib 1.3.1

```
======================================================================
ERROR: statsmodels.graphics.tests.test_gofplots.TestProbPlotRandomNormalMinimal.test_probplot_other_prbplt
----------------------------------------------------------------------
Traceback (most recent call last):
  File "E:\Josef\testing\tox\py33\lib\site-packages\nose-1.3.0-py3.3.egg\nose\case.py", line 198, in runTest
    self.test(*self.arg)
  File "E:\Josef\testing\tox\py33\lib\site-packages\numpy\testing\decorators.py", line 147, in skipper_func
    return f(*args, **kwargs)
  File "E:\Josef\testing\tox\py33\lib\site-packages\statsmodels\graphics\tests\test_gofplots.py", line 66, in
test_probplot_other_prbplt
    other=self.other_prbplot)
  File "E:\Josef\testing\tox\py33\lib\site-packages\statsmodels\graphics\gofplots.py", line 394, in probplot
    **plotkwargs)
  File "E:\Josef\testing\tox\py33\lib\site-packages\statsmodels\graphics\gofplots.py", line 736, in _do_plot
    ax.plot(x, y, fmt, **kwargs)
  File "E:\Josef\testing\tox\py33\lib\site-packages\matplotlib-1.3.1-py3.3-win32.egg\matplotlib\axes.py", line 4147, in plot
    for line in self._get_lines(*args, **kwargs):
  File "E:\Josef\testing\tox\py33\lib\site-packages\matplotlib-1.3.1-py3.3-win32.egg\matplotlib\axes.py", line 319, in _grab_next_args
    for seg in self._plot_args(remaining, kwargs):
  File "E:\Josef\testing\tox\py33\lib\site-packages\matplotlib-1.3.1-py3.3-win32.egg\matplotlib\axes.py", line 307, in _plot_args
    seg = func(x[:, j % ncx], y[:, j % ncy], kw, kwargs)
  File "E:\Josef\testing\tox\py33\lib\site-packages\matplotlib-1.3.1-py3.3-win32.egg\matplotlib\axes.py", line 259, in _makeline
    self.set_lineprops(seg, **kwargs)
  File "E:\Josef\testing\tox\py33\lib\site-packages\matplotlib-1.3.1-py3.3-win32.egg\matplotlib\axes.py", line 200, in set_lineprops
    raise TypeError('There is no line property "%s"' % key)
TypeError: There is no line property "other"
```

**Comment by bashtage** [comment]
@josef-pkt Looks like this should be closed

**Comment by josef-pkt** [comment]
The merged fix in #1414  only disables the unit tests, but doesn't fix the underlying API inconsistency.

I'm closing this (for test failures), but added a link to this issue in #2183 SUMM issue
